### DISCUSSION TITLE
Brazilian Portuguese Regex Profile

### DISCUSSION THREAD

**Discussion by araujojr82** [KBlixt/subcleaner#41] [Show and tell]
Hey @KBlixt 
This is the regex I've been working on, I've tested it on my library and refine it as I could.
Probably could use improvement on the REGEX.

Please feel free to use and modify as you see fit.
[portuguese.zip](https://github.com/KBlixt/subcleaner/files/11501311/portuguese.zip)

**Comment by KBlixt** [comment]
Alright! Will take a look tomorrow 👍👍

**Comment by KBlixt** [comment]
Damn... Just had a quick peek at it, an absolute immense job you've made. How much time did it take you to do this?

I'll add it in the morning for sure. 

Let me know if you need it updated unless you want to make a pull request yourself?

**Comment by araujojr82** [comment]
Thank you, but I couldn't have done it without your amazing tool!

I think I spent 2 weeks, between scanning my library and adding more stuff from the warnings log to the purge section, especially some of the members' names. 
I tried to be as in-depth as I could to avoid false positives.

Please go ahead and do any necessary changes and add it to your project.

**Reply by KBlixt** [reply]
just added it  as is to the default regex profiles. Thank you so much, amazing work! 😄 

let me know if you want to update it!

**Reply by araujojr82** [reply]
Glad to help! And thanks again for your work!
